proto/bootz.proto

```diff
@@ -141,7 +141,11 @@ message BootstrapDataSigned {
 }
 
 message GetBootstrapDataResponse {
```

Comment:
Just my idle musing: Since the proto is not yet being used for any RPC exchanges, isn't it ok to just remove the field? Why keep the proto backwards compatible when it doesn't need to be yet? Is there a benefit to giving vendors the option to use a more insecure format rather than only supporting the secure one?

Reply:
All vendors are using this field for testing and also for the upcoming image drop. If we deleted the field now it would break all integration and would delay progress quite a bit.

I'd rather get the initial testing working and slowly guide vendors to using the new field. Obviously we should make that a priority to do before we start ramping up fully.

Reply:
okedoke~

It'd be good to send this to ckl@ to review if you haven't already~

Reply:
Sounds good, we actually got feedback from him in https://github.com/openconfig/bootz/pull/66 but I accidentally closed that PR so making this one instead!